modules/version/version.go

```diff
@@ -0,0 +1,62 @@
+package version
+
+import (
+	"fmt"
+	"os"
+)
+
+/*
+The constants in this file are auto-replaced with the actual values
+during the build of the module.
+*/
+
+var (
+	// Branch of the code
+	Branch = "{branch}"
+	// Revision of the code
+	Revision = "{revision}"
+	// Dirty flag shows if the binary is built from a
+	// repo with uncommitted changes
+	Dirty = "{dirty}"
+)
+
+// Version interface
+type Version interface {
```

Comment:
No sure we need an interface for this. What does it bring here ?

Reply:
Nothing actually, may be in case we have another way to implement the version in the future may be